packages/@aws-cdk/aws-ecs-patterns/lib/base/network-load-balanced-service-base.ts

```diff
@@ -148,16 +150,29 @@ export abstract class NetworkLoadBalancedServiceBase extends cdk.Construct {
    */
   public readonly desiredCount: number;
 
+  /**
+   * The Network Load Balancer for the service.
+   */
   public readonly loadBalancer: NetworkLoadBalancer;
 
+  /**
+   * The listener for the service.
+   */
   public readonly listener: NetworkListener;
 
+  /**
+   * The target group for the service.
+   */
   public readonly targetGroup: NetworkTargetGroup;
+
   /**
    * The cluster that hosts the service.
    */
   public readonly cluster: ICluster;
 
+  /**
+   * The log driver to use for logging if logging is enabled.
```

Comment:
I would actually update this as `The log driver to use for logging`. if a LogDriver is provided, then we will default to logging is enabled.